Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/SqlHelper.ts

```diff
@@ -77,6 +70,20 @@ export function referencedByDocumentSql(documentId: string): string {
 
 // SQL for inserts/updates/upserts
 
+/**
+ * Returns the SQL statement to insert a document into the existence table
+ * @param documentId the document being inserted
+ * @param existenceId the existence id being inserted, this may be the same as the docuementId
+ * @returns SQL query string to insert into the existence table
+ */
+export const existenceInsertSql = (documentId: string, existenceId: string): string =>
```

Comment:
Why are these new functions declared as `const` while others are `function`? Why not keep consistent?